tests/test_toxenv.py

```diff
@@ -416,7 +416,7 @@ def test_travis_ignore_outcome(self, tmpdir, monkeypatch):
         """Test ignore_outcome without setting obey_outcomes."""
         tox_ini = tox_ini_ignore_outcome
         with self.configure(
-            tmpdir, monkeypatch, tox_ini, travis_language='generic'
+            tmpdir, monkeypatch, tox_ini, travis_version='3.4'
```

Comment:
Out of curiosity, what was the reason behind this change?

Reply:
Because the `generic` language wasn't even being used in the tox config, so it turns out that it was relying on the broken behavior from #75. And that broken behavior was fixed as a side-effect of this Pull Request, so this needed to be changed to match.